README.md

```diff
@@ -1,8 +1,313 @@
+  export GITEA_PASSWORD=$(kubectl get Secret -n fuseml-workloads gitea-creds -o jsonpath="{.data.password}" | base64 -d)
+  ```
+  It is possible to use external gitea server instead. Make sure to provide correct environment variables.
+  
+  `TEKTON_DASHBOARD_URL` is the path to the Tekton server. As with other components, Tekton is installed by `fuseml-installer` into your cluster. To get the right URL, call
+  ```bash
+  export TEKTON_DASHBOARD_URL=$(kubectl get VirtualService -n tekton-pipelines tekton -o jsonpath="{.spec.hosts[0]}")
+  ```
+
+  Now it's possible to execute `bin/fuseml_core`.
+  Use the `--help` flag to get the command line options that you can supply. By default the server listens on the follwing ports: 8000 (http) and 8080 (grpc)
+
+* Run the client
+
+  Executing the client with `--help` option will show the usage instuctions
+```bash
+    > bin/fuseml --help
+bin/fuseml is a command line client for the FuseML API.
+
+Usage:
+    fuseml [-host HOST][-url URL][-timeout SECONDS][-verbose|-v] SERVICE ENDPOINT [flags]
+
+    -host HOST:  server host (dev). valid values: dev, prod
+    -url URL:    specify service URL overriding host URL (http://localhost:8080)
+    -timeout:    maximum number of seconds to wait for response (30)
+    -verbose|-v: print request and response details (false)
+
+Commands:
+    application (list|register|get|delete)
+    codeset (list|register|get|delete)
+    runnable (list|register|get)
+    workflow (list|register|get|assign|list-runs)
+
+Additional help:
+    bin/fuseml SERVICE [ENDPOINT] --help
+
+Example:
+    bin/fuseml application list --type "predictor" --workflow "mlflow-sklearn-e2e"
+    bin/fuseml codeset list --project "mlflow-project-01" --label "mlflow"
+    bin/fuseml runnable list --id "ml-trainer-123" --kind "trainer" --labels '{
+          "function": "predict|train",
+          "library": "pytorch"
+       }'
+    bin/fuseml workflow list --name "workflow"
+```
+
+  Instead of providing `-url` argument to each command call, save the service URL into `FUSEML_SERVER_URL` enviornment variable and export it. Use this command to fill the variable with the URL of the `fuseml-core` server service that is installed in your Kubernetes cluster:
+
+  ```bash
+  export FUSEML_SERVER_URL=$(kubectl get VirtualService -n fuseml-core fuseml-core -o jsonpath="{.spec.hosts[0]}")
+  ```
+
+  If neither `FUSEML_SERVER_URL` nor `-url` value is set, client tries to connect to the server running locally at your machine.
+
+
+  From the usage output you can see that with the client you can operate with various FuseML artefacts.
+
+  * Codesets describe the code of your ML application, for example MLflow project. They are currently implemented as git repositories.
+
+    Create new codeset by calling `bin/fuseml codeset register` command. use the `--help` option to get the description of the command usage.
```

Comment:
`... a new codeset by calling the`?

Regarding the help it might be better to just mention at the beginning of this section that you can use `--help` for each command to get more details.

Reply:
ok